config/initializers/resque_config.rb

```diff
@@ -3,4 +3,4 @@
 Resque.redis = Redis.new(host: config[:host], port: config[:port], thread_safe: true)
 
 Resque.inline = Rails.env.test?
-Resque.redis.namespace = "heliotrope:#{Rails.env}"
+Resque.redis.namespace = "#{Settings.resque_namespace}:#{Rails.env}"
```

Comment:
Rails.env shouldn't be required here.  It will pull from whatever you put in the config/settings/production.yml file, and that file can be tailored per deployment.  I would also add a default with ||
```
Resque.redis.namespace = Settings.resque_namespace || "heliotrope:defaulty"
```

but you might want to open up pry somewhere and see what values that aren't specified in settings ends up being.  e.g. Settings.foo.   I'm assuming it's nil or something falsy, but I may be unhappily surprised here.